src/arc.ts

```diff
@@ -1,29 +1,54 @@
 import { Arc } from "@daostack/client";
+import Util from "lib/util";
 import { Observable } from "rxjs";
 
 const Web3 = require("web3");
 
+const INFURA_KEY = "16bDz7U53RbXysQiYOyc";
+
+const providers = {
+  dev: {
+    graphqlHttpProvider: "http://127.0.0.1:8000/subgraphs/name/daostack",
```

Comment:
I have been putting these global config variables in env vars too, so we can set them on Heroku. See webpack.dev.config.js in the DefinePlugin section

Reply:
Okay. Always a difficult call to decide where to put these kind of things, but environment variables seem a good place. Mind postponing the decision a bit still, also in the light of a possible change of deployment platform?